buildLogic/multiplatform-convention/src/main/kotlin/app/cash/sqldelight/multiplatform/MultiplatformConventions.kt

```diff
@@ -0,0 +1,74 @@
+package app.cash.sqldelight.multiplatform
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
+import org.jetbrains.kotlin.gradle.dsl.kotlinExtension
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import org.jetbrains.kotlin.konan.target.HostManager
+
+class MultiplatformConventions : Plugin<Project> {
```

Comment:
Out of curiosity, why not go with a precompiled script plugin?

Reply:
i think thats what this is, just not as a script, no?

Reply:
I think they call these binary plugins. A script plugin looks like a regular build.gradle.kts file e.g. https://github.com/eygraber/portal/blob/master/gradle-plugins/src/main/kotlin/portal-android-library.gradle.kts

Reply:
yea i mean I'll take normal kotlin over `.gradle.kts` or groovy any day of the week, I assume they both get precompiled so I'm not taking any preformance hits by doing it this way